db/migrate/20230522131908_add_library_tube_barcode_to_pacbio_run.rb

```diff
@@ -14,7 +14,7 @@ def down
       t.remove :pac_bio_library_tube_barcode
     end
     # Here we add a placeholder value in case any plate barcodes were added as null since the migration
-    PacBioRun.where(plate_barcode: nil).each do |run|
```

Comment:
I would add 'db/migrate/*' to the rubocop ignore list as we don't need to bother updating old migrations.

Reply:
Agreed. I will leave the change but add it to the gitignore.